test/controllers/billing-accounts.controller.test.js

```diff
@@ -8,10 +8,22 @@ const Sinon = require('sinon')
 const { describe, it, beforeEach, afterEach } = exports.lab = Lab.script()
 const { expect } = Code
 
+// Things we need to stub
+const ChangeAddressService = require('../../app/services/billing-accounts/change-address.service.js')
```

Comment:
Does the order need to be alphabetical for the 'things we need to stub' list? :)

Reply:
Yes!